uwblueprint/project-read-frontend
# Refactor theming and disable registration table text wrap

### Implementation description

some refactoring for:
- app theming
- registration table data generation



### Steps to test

1. yarn start
2. verify that column text does not wrap

<img width="1535" alt="Screen Shot 2021-04-05 at 7 27 38 PM" src="https://user-images.githubusercontent.com/37346399/113638500-fe1a2180-9644-11eb-8f76-accbe266fcc9.png">



### What should reviewers focus on?

- extensibility

### Checklist
